client/src/components/ProtectedRoute.js

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { withPageContext } from '../components/pageContext';
+
+class ProtectedRoute extends Component {
+
+    handleUnAuth = (activateToast, location) => {
+        activateToast('that was a fail', 'error')
```

Comment:
Maybe change the text message to "You need to be logged in to view this page." or something

Reply:
Good point. Done